MapboxNavigation/NavigationMapView.swift

```diff
@@ -1115,45 +1115,45 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
 @objc(MBNavigationMapViewDelegate)
```

Comment:
This should be MLNavigationMapViewDelegate as well?

Reply:
Probably the same should apply to MBNavigationMapViewCourseTrackingDelegate?

Reply:
Ok, that slipped through. The same problem exists in countless other places. Where I'm not sure: We have swapped the prefix `MGL` for `MLN` SDK-wide. But for the @objc attribute the prefix `MB` was used. Should we then change this to `ML` or align the prefix and use `MLN` here as well?

Reply:
I think it would make sense to swap out MBGL to MLN. We could do this in this PR or do this in a follow-up. I think only renaming a smaller part would be a bit weird.

I would be fine with merging this PR sooner rather than later and do a second renaming, maybe after #24 is merged as well to avoid too many conflicts?

Reply:
`MGL` to `MLN` is not the problem. I was referring to the `@obj` attributes which have the prefix `MB` with 98 occurencies in 41 files:

<img width="423" alt="image" src="https://github.com/maplibre/maplibre-navigation-ios/assets/895046/a89fa6c8-5712-451f-9d53-1728b4d66fdd">

Is there a problem renaming them to `MLN` as well? 

We can also wait until https://github.com/maplibre/maplibre-navigation-ios/pull/24 is merged, so I can resolve the conflicts before merging this one.